web-common/src/features/dashboards/time-series/utils.ts

```diff
@@ -3,7 +3,8 @@ import { bisectData } from "@rilldata/web-common/components/data-graphic/utils";
 import { roundToNearestTimeUnit } from "./round-to-nearest-time-unit";
 import { getDurationMultiple, getOffset } from "../../../lib/time/transforms";
 import { removeZoneOffset } from "../../../lib/time/timezone";
-import { TimeOffsetType } from "../../../lib/time/types";
+import { TimeOffsetType, TimeGrain } from "../../../lib/time/types";
+import { DateTime, DateTimeUnit } from "luxon";
```

Comment:
Avoid using luxon directly. We have most functions exposed in `/time/transforms`